edgexfoundry/go-mod-core-contracts
# fix: Remove redundant URL escaping from Client lib

- Remove redundant URL escaping from Client lib
- Refactor the HTTP request creating func

Close #730

Signed-off-by: bruce <weichou1229@gmail.com>







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-core-contracts/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **not impact the test**
  <link to docs PR>

## Testing Instructions

1. Run go-mod-core-contract unit test
2. Deploy EdgeX
3. Deploy simple device service
4. Test Set and Get command to verify the general function works.

## New Dependency Instructions (If applicable)
